lib/eunit/src/eunit_data.erl

```diff
@@ -573,22 +574,24 @@ get_module_tests(M) ->
 	    throw({module_not_found, M})
     end.
 
-get_module_tests_1(M, Es) ->
+get_module_tests_1(M, Es, Options) ->
```

Comment:
So I know this is not part of your change, just an observation that the very short variable names make the code hard to read I think. A few more characters that would make it easier to get the correct association without analyzing the context could be beneficial. It could perhaps be an enhancement if you think it is worth the time.

Reply:
I agree with observation. Unfortunately this is a style eunit code is written :-/ There is limited number of tests of application as well.

I will see what can be improved for eunit_data module - assuming some time box for activity.

Reply:
I've pushed a [commit](https://github.com/erlang/otp/pull/6157/commits/dd366cb65accf1b257f73fcbb1b1cfd572db68de) please have a look and let me know what you think
IMHO, result is a better clarity of the code but it is costly to implement and then review.